crates/db-arch/src/x86_64/cpuid/transformer/common.rs

```diff
@@ -34,31 +35,36 @@ pub fn update_feature_info_entry(
             u32::from(vm_spec.threads_per_core * vm_spec.cores_per_die * vm_spec.dies_per_socket),
         );
 
-    // A value of 1 for HTT indicates the value in CPUID.1.Ebx[23:16]
-    // (the Maximum number of addressable IDs for logical processors in this package)
-    // is valid for the package
+    // EDX bit 28: Hyper-Threading Technology, PAUSE. A value of 1 for HTT indicates the value in
```

Comment:
What does `PAUSE` here mean?

Reply:
It should be "PAUSE" instruction, but not sure:)